### ISSUE TITLE
Put "common" operations on top in stack editor

### ISSUE THREAD

**Issue by chregu** [rokka-io/rokka-dashboard#115]
Currently, the stack operations list is ordered alphabetically. And 'cause of that, there's for example the quite obscure "addframes" and "alpha" operations on top. Maybe we should put the common ones (like resize and crop) on top and the rest alphabetically. Of course needs some "hand crafting", but would help the enduser

**Comment by mweibel** [comment]
Yeah, that makes sense I guess. Or maybe have the list but before the select a bunch of "buttons" with the most used ones? 
@pedro-couto do you have a good idea?
